mpd/server.py

```diff
@@ -133,16 +138,19 @@ def update_db(mpd_root):
         with open(os.path.join(_BLISSIFY_DATA_HOME, "latest_mtime.txt"), "r") as fh:
             latest_mtime = int(fh.read())
     except FileNotFoundError:
-        pass
+        logging.error("latest_mtime.txt file not found. Please call --full-rescan before --update.")
+        return
     songs = [x["file"] for x in client.find("modified-since", latest_mtime)]
     subprocess.check_call(["blissify", mpd_root] + songs)
     for song in songs:
-        last_modified = client.find("file", song)["last_modified"]
+        close_connection(client)
+        client = init_connection()
```

Comment:
Ok, this should be refactored anyways #11 

We can stick with this and edit it later.